R/calendar_range.R

```diff
@@ -16,12 +16,15 @@
 #' @param end_placeholder Text visible in the end calendar input when nothing is inputted.
 #' @param min Minimum allowed value in both calendars.
 #' @param max Maximum allowed value in both calendars.
+#' @param start_label Label text in the start calendar.
+#' @param end_label Label text in the end calendar.
 #'
 #' @rdname calendar_range
 #' @export
 #'
 calendar_range <- function(input_id, type = "date", start_value = NULL, end_value = NULL,
-                           start_placeholder = NULL, end_placeholder = NULL, min = NA, max = NA) {
+                           start_placeholder = NULL, end_placeholder = NULL, min = NA, max = NA,
+                           start_label = NULL, end_label = NULL) {
```

Comment:
I've checked that in the majority of inputs we keep the order of arguments the same as in basic shiny inputs, this is: id, label, value/choices. How about putting the labels after the input_id argument?